src/build/upstream/build-data-from-upstream.ts

```diff
@@ -0,0 +1,29 @@
+import { sliceUpstreamGamemaster } from './slice-upstream-gamemaster'
+import { transformPvpokePokedexEntry } from './transform-pvpoke-pokedex-entry'
+import { transformPvpokeMove } from './transform-pvpoke-move'
+import { writeObjAsJson } from './util'
+
+export const buildDataFromUpstream = async () => {
+  const { 
+    pokemon: pvpokePokemon, 
+    moves: pvpokeMoves 
+  } = await sliceUpstreamGamemaster()
+
+  await writeObjAsJson( 
```

Comment:
We keep both copies, pvpoke style and internal style, so we can run comparisons etc